src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -14,6 +14,7 @@
 
     <!-- The test is looking for debugger attributes we would have stripped with NativeAOT -->
     <IlcKeepManagedDebuggerSupport>true</IlcKeepManagedDebuggerSupport>
+    <UseLocalAppHostPack Condition="'$(TestReadyToRun)' == 'true'">true</UseLocalAppHostPack>
```

Comment:
This doesn't look like a per-test setting - should this go to a place where we define what `TestReadyToRun` means in general?